solana-programs/anchor/audius-data/cli/main.ts

```diff
@@ -294,7 +294,7 @@ program
   .option("--num-playlists <integer>", "number of playlists to generate")
   .option("--id <integer>", "ID of entity targeted by transaction")
   .option("-sp-id, --cn-sp-id <string>", "ID of incoming content node")
-  .option("-ci, <boolean>", "set to true to seed content node wallet and pkey with dummy values", false)
+  .option("--deterministic, <boolean>", "set to false to seed content node wallet and pkey dynamically from local env (only when running ganache without --deterministic)", true)
```

Comment:
Is there a way for me using the CLI to specify the following fields for `initContentNode` - spID, ownerWallet (eth pubkey) and authority (sol pubkey)? I think the work here looks great but this is the real functionality we need in addition to local stuff

Reply:
I'll submit a fast-follow PR for this and other fixes

Reply:
no worries, it's a relatively trivial change after this great PR - can do so after ID related work